Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "regorus"
+description = "A fast, lightweight Rego (OPA policy language) interpreter"
```

Comment:
Does crates.io allow marking a release as alpha/beta?

Reply:
Yes, we can add `-alpha` suffix, I'll change it to `0.0.1-alpha`